src/apps/activities/domain/response_values.py

```diff
@@ -62,13 +62,13 @@ class ABTrailsValues(PublicModel):
 class _SingleSelectionValue(PublicModel):
     id: str | None = None
     text: str
-    image: str | None
-    score: int | None
-    tooltip: str | None
+    image: str | None = None
+    score: int | None = None
+    tooltip: str | None = None
     is_hidden: bool = Field(default=False)
-    color: Color | None
-    alert: str | None
-    value: int | None
+    color: Color | None = None
+    alert: str | None = None
+    value: int | None = None
```

Comment:
Are you sure that this is legal? :)
As i know `value` attribute works as mapper key for answers on frontend

Reply:
agreed, value seems have to be required. Please check with FE team